### PR TITLE
ccadb2OneCRL: preserve leading zeros on serial numbers

### LINKED ISSUE #190: ccadb2OneCRL is dropping the leading zeros when adding entries to OneCRL
ccadb2OneCRL is apparently dropping the leading zeros when adding entries to OneCRL. This causes us to have to go back later and manually remove the incorrect entry and manually add the correct entry.

Bugs where we've seen this:
https://bugzilla.mozilla.org/show_bug.cgi?id=1717100
https://bugzilla.mozilla.org/show_bug.cgi?id=1694779

### PR DESCRIPTION
**Pull request by jschanck**
Resolves #190

### CODE REVIEW

File: ccadb2OneCRL/utils/utils_test.go [hunk 1]
```diff
@@ -20,6 +20,20 @@ func assertBytes(t *testing.T, data string, expected []byte) {
 	}
 }
 
+func TestRawSerialBytes(t *testing.T) {
+	// DER for the version and serialNumber fields of a tbsCertificate.
+	tbsCertPrefix := []byte{
+		0x30, 0x0f,
+		0xa0, 0x03, 0x02, 0x01, 0x02,
+		0x02, 0x08, 0x00, 0xaa, 0xaa, 0xaa, 0xaa, 0xaa, 0xaa, 0xaa,
```
Comment (mozkeeler) [inline ccadb2OneCRL/utils/utils_test.go:28]: Might be good to have some more testcases like if there isn't a leading zero, if there's a leading `0xff`, etc.

File: ccadb2OneCRL/utils/normalize.go [hunk 2]
```diff
@@ -5,13 +5,30 @@
...
 	"github.com/pkg/errors"
 )
 
+type tbsCertWithRawSerial struct {
+	Raw          asn1.RawContent
+	Version      asn1.RawValue `asn1:"optional,explicit,default:0,tag:0"`
+	SerialNumber asn1.RawValue
+}
+
+// Extract the raw bytes of the serial number field from a tbsCertificate..
```
Comment (mozkeeler) [inline ccadb2OneCRL/utils/normalize.go:22]: tiny nit: extra `.` at the end

**Review by mozkeeler — DISMISSED** [review]
This looks good to me. I left some minor comments. Unfortunately, I don't know how to run this or update it in production.

**Comment by WilsonKathleen** [comment]
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1778598 to request help with this.

**Comment by mozkeeler** [comment]
@smarnach I gave you admin access to this repo.

**Comment by smarnach** [comment]
Thanks! It looks like the integration between Circle CI and GitHub got "stuck" in some way. We've seen that for a few repos when SAML was enabled for the mozilla org. The easiest way to fix this is to disable Circle CI for this project, and then immediately re-enable it. This will remove all "followers" of this project on Circle CI. So if you currently follow the project, and still want to do so in the future, you have to click the follow button in Circle CI again once I'm done.

**Pull request closed without merging**